standards/documenting-infrastructure-owners.md

```diff
@@ -0,0 +1,28 @@
+---
+category: Operating services
+---
+# Documenting owners of infrastructure
+
+All of your infrastructure resources should be tagged with details of its owners, what service it is part of, and what type of environment it is.
+
+Without this, teams responsible for supporting your services (security, finance, and web operations) may not be able to keep your infrastructure secure, paid for by the right people, and appropriately supported. Documenting this on the infrastructure itself ensures that it's as accurate and up-to-date as possible.
+
+You should do this even if you're managing infrastructure in your own account: other teams may end up needing access to that account, or may receive access from another account.
+
+## Tagging your infrastructure
+
+All of our existing cloud hosting providers support tagging ([AWS](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html), [Azure](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-using-tags), [vCloud](https://blogs.vmware.com/vsphere/2012/03/creating-custom-metadata-using-the-vcloud-api.html)). If your infrastructure is defined in code ([as it should be](https://www.gov.uk/service-manual/technology/manage-your-software-configuration#use-infrastructure-as-code)), you can probably specify your tags in that code.
+
+## Tags you should use
+
+To ensure we can consistently search for, and report on, the tags we use, you should use the following tags. In all values, only use acronyms if you're confident that someone from another part of government would understand them.
+
+- `business-unit`: Should be one of `HQ`, `HMPPS`, `OPG`, `LAA`, `HMCTS`, or `CICA`. If none of these are appropriate, use an appropriate name for the area of the MOJ responsible for the service.
+- `application`: Should be the full name of the application or service (and acronym version, if commonly used), e.g. `Prison Visits Booking`, `Claim for Crown Court Defence (CCCD)`.
```

Comment:
@glenjamin Also suggested this should possible be `service`. I opted for `application` assuming there might be more here than just traditional services, but I'm thinking maybe `service` is better. Does anyone have thoughts either way?